app/src/main/res/layout/devs_list_item.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <RelativeLayout
+        android:id="@+id/image_wrapper"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:padding="@dimen/list_item_margin">
+
+        <ImageView
+            android:id="@+id/dev_list_image"
+            android:src="@mipmap/ic_launcher"
+            android:layout_width="@dimen/list_item_image_width"
+            android:layout_height="@dimen/list_item_image_width" />
+    </RelativeLayout>
+
+     <TextView
+        android:text="@string/app_name"
```

Comment:
Don't you think that the id `app_name` doesn't correspond to the purpose of this TextView? If it's supposed to hold the name of a developer, perhaps you should use something like `dev_name`.

Reply:
I had used the `app_name` in the text view and `ic_launcher` in the image view as placeholders. If they cause confusion then I guess I should leave them empty or create dedicated values for them. on it.